test_utils/os_utils.py

```diff
@@ -8,9 +8,9 @@
 import posixpath
 import re
 import time
-from datetime import timedelta, datetime
 
-from aenum import IntFlag, Enum, IntEnum
+from datetime import timedelta, datetime
+from aenum import IntFlag, Enum, IntEnum, StrEnum
```

Comment:
```aenum```
Can`t we just use enum here?

Reply:
indeed current enum version incorporates types that were previously present only in aenum

Reply:
it's outdated, yesterday I changed it to enum :)

Reply:
I've added what I meant in a new comment :)